packages/light-api/src/srml/balances.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017-2018 @polkadot/light-api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import ApiRx from '@polkadot/api/rx';
+import { AccountId } from '@polkadot/types';
+import { switchMap } from 'rxjs/operators';
+
+import { NODES } from '../LightApi';
+import { ReactiveGraph } from '../ReactiveGraph';
+import { ANY_VALUE } from '../types';
+import { addEventNode } from '../utils/addEventNode';
+
+const SECTION = 'balances';
+
+export function transferFee (graph: ReactiveGraph, api: ApiRx) {
+  const myNode = `query.${SECTION}.transferFee()`;
```

Comment:
I guess in the future we'll first want to check if the node already exists, and if so return it? (and do this for every node function) like it's done in addEventNode

Reply:
Yes, exactly. This will replace memoization in light.js